ui/components/ClusterOverview/components/TopicsTable.tsx

```diff
@@ -5,8 +5,58 @@ import { Link } from "@/i18n/routing";
 import { Truncate } from "@patternfly/react-core";
 import { TableVariant } from "@patternfly/react-table";
 import { useTranslations } from "next-intl";
+import { ReactNode } from "react";
+import { Icon } from "@/libs/patternfly/react-core";
+import {
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  ExclamationTriangleIcon,
+} from "@/libs/patternfly/react-icons";
+import { TopicStatus } from "@/api/topics/schema";
 
-export const TopicsTableColumns = ["name", "cluster"] as const;
+export const TopicsTableColumns = ["name"] as const;
+const StatusLabel: Record<TopicStatus, ReactNode> = {
```

Comment:
Can these labels be removed too?

Reply:
I didn't notice it. Thanks for pointing it out! I have removed it.